pygmt/datasets/samples.py

```diff
@@ -343,3 +346,17 @@ def load_mars_shape(**kwargs):
         fname, sep="\t", header=None, names=["longitude", "latitude", "radius(m)"]
     )
     return data
+
+
+def _load_static_earth_relief(**kwargs):
+    """
+    Load the static_earth_relief file for internal testing.
+
+    Returns
+    -------
+    data : xarray.DataArray
+        A grid of Earth relief for internal tests.
+    """
+    fname = which("@static_earth_relief.nc", download="c")
+    data = xr.open_dataarray(fname)
+    return data
```

Comment:
```suggestion
    return xr.open_dataarray(fname)
```
In addition to consolidating these two lines, I think you should consider using `from pygmt.io import load_dataarray` rather than `import xarray as xr` and `load_arrayarray(fname)` rather than `xr.open_datarray(fname)`. Right now, these two options return the same ouput. However, we could extend the load_dataarray function to add other important metadata (e.g., citations) from the NetCDF global attributes to xarray.DataArray.attrs whereas open_dataarray is less flexible for future enhancements.

Reply:
I'll switch the function to use `load_dataarray`.